docs/data-sources/cce_cluster.md

```diff
@@ -69,11 +69,15 @@ In addition to all arguments above, the following attributes are exported:
 
 * `enterprise_project_id` - The enterprise project id of the cce cluster.
 
-**endpoints**
+* `endpoints` - The access addresses of kube-apiserver in the cluster.
 
-* `internal` - The address accessed within the user's subnet.
+* `endpoints/url` - The URL of the cluster access address.
```

Comment:
suggest to use the format of nested block:
```Structure is documented below.```

Reply:
fixed

Reply:
fixed